### ISSUE TITLE
[Enhancement] Android Wear Support

### ISSUE THREAD

**Issue by supernoctem** [raatmarien/red-moon#69]
Android Wear support for watches, to set the same screen filters on the watch.

**Comment by Kasijjuf** [comment]
I hope I'm wrong, but unfortunately I believe at this  time that Wear apps require Google Play Services and thus can't be done in a free/libre app. Someone please tell me I'm wrong on this one, cause I would love to be wrong.

**Comment by supernoctem** [comment]
From some quick searching, I can see that sending settings and things to Android Wear might be difficult without Play Services. The only other way would be using the standard bluetooth api, but I have no idea how this work.

[1] https://stackoverflow.com/questions/30063783/android-wear-without-google-play-services
[2] https://developer.android.com/training/wearables/apps/creating.html

**Comment by smichel17** [comment]
Maybe the best solution for the meantime would be to offer Wear support through an add-on that uses play services but is distributed in its own apk.

**Comment by raatmarien** [comment]
I agree with @smichel17 that an add-on using Google Play Services would probably be the best solution. However, I don't own an Android Wear devices, so I can not develop this add-on.

**Comment by Jawnnypoo** [comment]
If you are worried about it being free with reguards to submitting it to F-Droid, the usual solution is to have multiple build flavors, one that is "free" containing only open FOSS, and then another flavor that contains things like Google Play Services, Fabric, etc.

**Comment by Kasijjuf** [comment]
@Jawnnypoo, The problem with that is that since this app is licensed under GPLv3+, unless you can get Google to release Play Services under a GPLv3-compatible free software license (good luck with that), Play Services may not be included in the project.

**Comment by mirh** [comment]
You aren't linking or including play services in your program. Just its API (if any). 
That's quite different: for as much as you (and the app) know it could even be as well you are talking with [MicroG](https://github.com/microg/android_external_GmsLib/tree/master/play-services-wearable). 

Don't confuse *being FOSS* and *being able to work in a totally FOSS environment* (such as F-droid). Of course the later is preferable ([ideas](https://github.com/microg/android_external_Wearable)), but god, it's not like telegram or... just anything on windows don't do this every time?

**Comment by smichel17** [comment]
> Don't confuse being FOSS and being able to work in a totally FOSS environment

@mirh That's fair. However, I have no interest in putting in developing or maintaining Red Moon for a non-[FLO](https://wiki.snowdrift.coop/about/free-libre-open) environment. If someone else wants to do that, I'll do my best to stay out of their way, but that's about it.

**Comment by mirh** [comment]
Well, the only FLO environment that will get to happen if any ever, is microg advancing and all (they *already* might have a good enough implementation, idk)
Which nevertheless will still necessarily have to mimic google's standards api. 

So I dunno, it all seems about whenever best you want to allocate your time.

**Comment by smichel17** [comment]
Yeah. It's not that I see *no* benefit, but there's other features (and other projects) I want to spend time on, too. If Wear support ever becomes possible in a completely FLO environment, my priorities might change;  I just wanted to set expectations in the meantime (also reflected by the tags/milestone).
